src/arrayMethodSort.js

```diff
@@ -5,8 +5,25 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
you can set just default value for compareFunction, so it will make a code shorter 
```suggestion
  [].__proto__.sort2 = function(compareFn = (a, b) => `${a}` > `${b}`) {
```